modules/ding_ddbasic/plugins/content_types/search_extended.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @file
+ * Content type plugin for all opening hours.
+ */
+
+/**
+ * Array describing the plugin to CTools/Panels.
+ */
+$plugin = array(
+  'title' => t('Search form: extended'),
+  'description' => t('Show todays opening hours for all libraries.'),
```

Comment:
Is this a wrong description? I don't think opening hours is related to extended search.